Formula/datafusion.rb

```diff
@@ -16,8 +16,14 @@ class Datafusion < Formula
   end
 
   depends_on "rust" => :build
+  # building ballista requires installing rustfmt
+  depends_on "rustup-init" => :build
```

Comment:
This seems like the wrong way to go about it. We should instead add a `rustfmt` formula and make that a build dependency here.

Reply:
ok let me try to do that

Reply:
Thanks. You can open a separate PR for `rustfmt` to keep things simple here.

Reply:
let me try that in https://github.com/Homebrew/homebrew-core/pull/89599

Reply:
thanks for the advice, this is now updated (and with minimal changes thanks to rustfmt)